contracts/src/chainlink/cairo/ocr2/aggregator.cairo

```diff
@@ -1109,10 +1067,19 @@ end
 
 # --- Transmitter Payment
 
-func calculate_reimbursement() -> (amount : felt):
-    # TODO:
-    let amount = 0
-    return (amount)
+const MARGIN = 115
```

Comment:
Not sure how useful is the fixed margin as we can scale dynamically via `billing.gas_base|gas_per_signature` configuration.